components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/editor/commons/js/utils/utils.js

```diff
@@ -15,6 +16,110 @@ define(['require', 'jquery'],
+                    name: "ex4aerertrtrt", status: "partially-installed",
+                    info: {
+                        description: " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi"
+                        ,
+                        install: "To install this ex4 extension you have to set  all dependency of it" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi" + " this ex4 extension gives the string conversion features to Siddhi" +
+                            " app.This ex4 extension gives the string conversion features to Siddhi"
+                        ,
+                    }
+                },
+                {name: "kafka", status: "not-installed"},
```

Comment:
is this json has the latest format ? @senthuran16 please check

Reply:
no its not a latest format i will change this json data as @senthuran16  shared json data.

Reply:
88cbc98 this commit will fix this change request